dspace-api/src/main/resources/spring/spring-dspace-addon-import-services.xml

```diff
@@ -53,6 +56,15 @@
           class="org.dspace.importer.external.pubmed.metadatamapping.PubmedFieldMapping">
     </bean>
 
+    <bean id="RisImportService"
+          class="org.dspace.importer.external.ris.service.RisImportMetadataSourceServiceImpl" scope="singleton">
```

Comment:
Scope singleton is the default, you can leave out the initial scope declaration.